test/testtask_test.rb

```diff
@@ -25,4 +26,12 @@ def test_define_rake_tasks
   def test_composing_rake_testtask
     assert_equal @rake_task, @ttnt_task.rake_testtask
   end
+
+  def test_expanded_file_list
+    # It gathers tests from both `pattern` and `test_files` option for Rake::TestTask
+    assert_equal(
```

Comment:
You can certainly avoid the call on several lines with:

~~~ruby
test_files = Rake::FileList['test/**/*_test.rb'] + ['test/dummy_test.rb']
assert_equal test_files, @ttnt_task.expanded_file_list
~~~